docs/quickstarts/github.rst

```diff
@@ -42,25 +42,28 @@ Code
     If you are running this code on Heroku, you'll need to use the
     :class:`werkzeug.contrib.fixers.ProxyFix` middleware. See :doc:`../proxies`.
 
-When you run this code, you must set the :envvar:`OAUTHLIB_INSECURE_TRANSPORT`
-environment variable for it to work. For example, if you put this code in a
-file named ``github.py``, you could run:
+If you run this code locally or without HTTPS enabled (see warning below), 
+you must set the :envvar:`OAUTHLIB_INSECURE_TRANSPORT` environment variable
+to disable the HTTPS requirement imposed by ``oauthlib``. For example, if
```

Comment:
I expect that most people won't know what `oauthlib` is, and won't care. Can you change this to:

```
to disable the HTTPS requirement imposed by ``oauthlib``, which is part of Flask-Dance.
```